scripts/create-devices-inventory-from-cvp.py

```diff
@@ -68,7 +68,7 @@ def main() -> None:
     payload = json.dumps({"userId": args.username, "password": args.password})
     headers = {"Content-Type": "application/json", "Accept": "application/json"}
     response = requests.request(
-        "POST", URL, headers=headers, data=payload, verify=False
+        "POST", URL, headers=headers, data=payload, verify=False, timeout=10
```

Comment:
Probably should catch timeout error here with some "meaningful" message?

Reply:
I would let @ksator updating his script :)

This update is only because linting complained about this missing parameter in the CI

Reply:
OK I will do this later on